### PR TITLE
Feature/add trone and trduo

### PR DESCRIPTION
**Pull request by pl-kabaradjian**

### CODE REVIEW

File: README_duo.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+Using module
+============
+
+To use the ROS node you just need to:
+* Create a ROS Workspace
+* Copy the node terarangerduo package into the workspace src directory
+* Compile using: catkin_make 
+* Run using: rosrun terarangerduo terarangerduo_node _portname:=/dev/ttyUSB0
```
Comment (msadowski) [inline README_duo.md:14]: This probably needs updating to reflect the new naming (we can also be make it a part of the main readme)

File: README_one.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Using module
+============
+
+To use the ROS node you just need to:
+* Create a ROS Workspace
+* Copy the node terarangerone package into the workspace src directory
+* Compile using: catkin_make 
+* Setup environment: source devel/setup.sh
+* Run using: rosrun terarangerone terarangerone_node _portname:=/dev/ttyUSB0
```
Comment (msadowski) [inline README_one.md:15]: This probably needs updating to reflect the new name

File: include/teraranger/teraranger_duo.h [hunk 3]
```diff
@@ -0,0 +1,98 @@
+/****************************************************************************
```
Comment (msadowski) [inline include/teraranger/teraranger_duo.h:1]: I would remove those header strings, this info should be contained in package.xml (and maybe in a licence file)

File: include/teraranger/teraranger_duo.h [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+#ifndef TERARANGERDUO_TERARANGER_H_
```
Comment (msadowski) [inline include/teraranger/teraranger_duo.h:37]: Let's use` #pragma once` to be consistent with terraranger_array and other header files

File: include/teraranger/teraranger_one.h [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+#ifndef TERARANGERONE_TERARANGER_H_
```
Comment (msadowski) [inline include/teraranger/teraranger_one.h:36]: Let's use `#pragma once` and remove the doc string

File: src/teraranger_duo.cpp [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
```
Comment (msadowski) [inline src/teraranger_duo.cpp:36]: Let's remove this string too!

File: src/teraranger_one.cpp [hunk 7]
```diff
@@ -0,0 +1,203 @@
+/****************************************************************************
```
Comment (msadowski) [inline src/teraranger_one.cpp:1]: Remove

File: include/teraranger/teraranger_duo.h (hunk 3 above)
Comment (pl-kabaradjian) [reply include/teraranger/teraranger_duo.h:1]: done

File: include/teraranger/teraranger_duo.h (hunk 4 above)
Comment (pl-kabaradjian) [reply include/teraranger/teraranger_duo.h:37]: done

File: include/teraranger/teraranger_one.h (hunk 5 above)
Comment (pl-kabaradjian) [reply include/teraranger/teraranger_one.h:36]: done

File: src/teraranger_duo.cpp (hunk 6 above)
Comment (pl-kabaradjian) [reply src/teraranger_duo.cpp:36]: done

File: src/teraranger_one.cpp (hunk 7 above)
Comment (pl-kabaradjian) [reply src/teraranger_one.cpp:1]: done

File: src/teraranger_duo.cpp (hunk 6 above)
Comment (pl-kabaradjian) [reply src/teraranger_duo.cpp]: done

**Pull request merged**